web/src/project/ProjectSummary.tsx

```diff
@@ -455,7 +458,7 @@ export const ProjectSummary = () => {
     };
 
     const seqrLinks = () => {
-        if (!summary?.seqr_links.length) {
+        if (!summary?.seqr_links || _.isEmpty(summary?.seqr_links)) {
```

Comment:
Can you pass the seqr_links directly to isEmpty, removing the first check, eg:

```suggestion
        if (_.isEmpty(summary?.seqr_links)) {
```

Or use optional coalesce:

```suggestion
        if ((summary?.seqr_links?.length ?? 0) == 0) {
```

Reply:
I didn't think you could check object size with .length, that's why it broke in the first place. So now I check if the object doesn't exist or if it's empty, then don't render anything.

Reply:
But you can do that with the optional chain operator, eg: `summary?.length` will be falsey if empty or unset.

Reply:
Sorry my point is that objects don't have a length so {}.length is not a thing. I mistakenly added in .length in a previous PR and just wanted to fix it. So for objects, I want to check is it not null/undefined and is it non empty. I thought that best done with is !object || _.isEmpty(object)? Could use Object.keys(object).length too

Reply:
How about `if (!Object.keys(summary.seqr_links ?? {}).length) return <></>` . This catches undefined, null, nonexistent key and empty objects